share/completions/btrfs.fish

```diff
@@ -0,0 +1,134 @@
+# Completions for btrfs-progs
+# Author: Akatsuki (akiirui) <imaykiller@gmail.com>
+
+# Todo:
+# 1. Detailed options per command
+# 2. Simplified descriptions
+# ...and more
+
+# Filter the completions per command groups
+function __btrfs_options_groups
+    if test -z $argv
```

Comment:
This won't work, because `test` conforms to POSIX. That requires that `test` with any one argument return true, so `test -z $argv` with an empty $argv will expand to `test -z`, which needs to be true.

*ALWAYS* quote variables when used with `test`. Sorry for the inconvenience.

Reply:
Alternatively use `if not set -q argv[1]`.

Reply:
Thanks for your remind